### ISSUE TITLE
Change the "+ Install" icon so it resembles the app market

### ISSUE THREAD

**Issue by paulproteus** [sandstorm-io/sandstorm#2052]
Context:

- On `/apps` there is a `+` icon that indicates that you can install things.
- Maybe it'd be better to have it visually cue the app market.

I believe that @jacksingleton was excited to see us possibly do this. Anyway @neynah expressed an interest in designing it, so I'm filing this issue.

Not assigning her since I don't know if she was interested in doing it any time particularly soon; she can speak to that.

**Comment by zombiezen** [comment]
One thing I've done recently: I was trying to install a new app from the app market and hit the "Uninstall..." button.  Since install blends in with the rest of the app grid, the "Upload app..." and "Uninstall..." links actually jump out more to me because "Install..." is below the "All apps" fold.

**Comment by neynah** [comment]
Yeah it is pretty odd that installing apps ends up below the fold once the "Most used" section displays.
